Bottom: effd516d48f2cb19d399aca73a5732022e624cae
Top:    d86d3cc1f7428699bee8f676996fdf9b43888970
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 00:16:30 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 3468b9b..399b614 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
